index.js

```diff
@@ -221,6 +221,7 @@ Blake2b.prototype.digest = function (out) {
   assert(buf.length >= this.outlen, 'out must have at least outlen bytes of space')
   blake2bFinal(this, buf)
   if (out === 'hex') return hexSlice(buf)
+  assert(buf instanceof Uint8Array, 'input must be Uint8Array or Buffer')
```

Comment:
Can you move this up under the other assert? It's too late to assert it at that point since it has already been attempted to be written to :) Then this is good to merge!

Reply:
Fixed